apps/chef_test/src/chef_test_suite_helper.erl

```diff
@@ -32,10 +34,29 @@
          make_id/1,
          make_az_id/1,
          actor_id/0,
+
+         make_orgs/0,
+         org_name/0,
+         other_org_name/0,
+
          the_org_id/0,
-         other_org_id/0
+         other_org_id/0,
+
+         create_record/1,
+         list_records/1,
+         update_record/1,
+         fetch_record/1,
+         delete_record/1
         ]).
 
+-define(ORG_AUTHZ_ID, <<"10000000000000000000000000000002">>).
+-define(ORG_NAME, <<"testorg">>).
+
+-define(OTHER_ORG_AUTHZ_ID, <<"99999999999999999999999999999999">>).
```

Comment:
I'm pretty sure the policies case is unique or mostly unique in this regard, but I'm finding it helpful to have a handful of different authz_ids around. The reason it's useful for this case is that I have some functions that do multiple creates and also some of the DB rows denormalize Authz IDs of related rows to make the most common query cases most efficient. With different Authz IDs in the tests, I can be sure I put the right Authz IDs in the right record fields when creating records for the various kinds of DB queries (and if I get it wrong, the DB will return a FK error so I know something's wrong).